package.json

```diff
@@ -10,7 +10,11 @@
   "scripts": {
     "debug": "node-debug --hidden node_modules _mocha --watch",
     "debug-ide": "./bin/debug-ide",
-    "test": "mocha -R mocha-multi --reporter-options nyan=-,json=.results.json"
+    "copy-git-hooks": "./bin/copy-hooks",
+    "test": "mocha -R mocha-multi --require test/root.js --reporter-options nyan=-,json=.results.json",
+    "bundle": "browserify index.js -t babelify --exclude react/addons --exclude react/lib/ExecutionEnvironment --exclude react/lib/ReactContext -o bundle.js",
+    "bundle:test": "browserify test/index-test.js -t babelify --exclude react/addons --exclude react/lib/ExecutionEnvironment --exclude react/lib/ReactContext -o browser-test-bundle.js",
```

Comment:
Are these exclusions reliable? The only reason I ask is that it seems like the React team likes to move stuff around -- we'll have to update these lines if something gets out of whack here.